tests/server/api/conftest.py

```diff
@@ -40,7 +41,7 @@ async def span_data_with_documents(prod_db) -> None:
             events=[
                 {
                     "name": "exception",
-                    "timestamp": datetime.now().isoformat(),
+                    "timestamp": datetime.now(),
```

Comment:
This one intentional? Seems like the test should pass without this change.

Reply:
it was a stopgap because i didn't understand why it didn't work. but it works now because i added the json_serializer to the test engine (the production engine already has the serializer so it works fine in production)